src/ansys/geometry/core/connection/conversions.py

```diff
@@ -680,3 +689,37 @@ def line_to_grpc_line(line: Line) -> GRPCLine:
     start = line.origin
     end = line.origin + line.direction
     return GRPCLine(start=point3d_to_grpc_point(start), end=point3d_to_grpc_point(end))
+
+
+def grpc_material_to_material(material: GRPCMaterial) -> Material:
+    """Convert a material gRPC message to a ``Material`` class."""
+    properties = []
+    density = Quantity(0)
```

Comment:
You should probably add units here (kg / m^3)
```suggestion
    density = 0 * ( UNITS.kg / UNITS.m**3 )
```

You will also have to add the ``from ansys.geometry.core.misc import UNITS`` statement.